test/test_va_api_fixture.cpp

```diff
@@ -189,6 +236,56 @@ void VAAPIFixture::doFillConfigAttribList()
     EXPECT_EQ(m_configAttribList.size(), m_vaConfigAttribs.size());
 }
 
+void VAAPIFixture::doTestCreateImage(const uint32_t& currentFmt,
+		const std::pair<uint32_t, uint32_t>& currentResolution)
+{
+    VAStatus status;
+    VAImageFormat currentImageFormat;
+
+    currentImageFormat.fourcc = currentFmt;
+
+    status = vaCreateImage(m_vaDisplay, &currentImageFormat,
+		 currentResolution.first, currentResolution.second, &m_image);
+
+    if (status == VA_STATUS_SUCCESS)
+	ASSERT_ID(m_image.image_id);
+    else
+	EXPECT_TRUE(
+		status == VA_STATUS_ERROR_OPERATION_FAILED);
+}
+
+bool VAAPIFixture::doCompareFormat (const VAImageFormat& imageFormat,
+					        const uint32_t& current)
+{
+        return (imageFormat.fourcc == current);
+}
+
+uint32_t VAAPIFixture::doFindImageFormatInList (uint32_t format)
+{
+    if(std::find_if(m_imageFmtList.begin(), m_imageFmtList.end(),
+        std::bind(doCompareFormat, std::placeholders::_1, format))
+    != m_imageFmtList.end() )
+        return format;
+    else
+	return 0;
+}
+
+bool VAAPIFixture::doCompareSubPicFormat(const VAImageFormat& subPicImagefmt,
+					            const uint32_t& subpicfmt)
+{
+        return (subPicImagefmt.fourcc == subpicfmt);
+}
+
+uint32_t VAAPIFixture::doFindSubPicImageFormatInList(uint32_t subPicFmt)
+{
+    if(std::find_if(m_subPicFmtList.begin(), m_subPicFmtList.end(),
+        std::bind(doCompareSubPicFormat,  std::placeholders::_1, subPicFmt))
+		 != m_subPicFmtList.end() )
```

Comment:
283 - This line is part of the if and should be indented to reflect that, lining up with the line above.
Hard tabs here instead of soft. Lines 283, 284, and 286.

Reply:
Fixed.